abcpy/modelselections.py

```diff
@@ -0,0 +1,259 @@
+from abc import ABCMeta, abstractmethod
+import numpy as np
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+
+
+
+class ModelSelections(metaclass = ABCMeta):
+    """This abstract base class defines a model selection rulr of how to choose model from a set of models
```

Comment:
It's difficult to explain the high level goal. In general this should be known to a practitioner who wants to perform modelselection.